dev/source/docs/SITL_simulation_parameters.rst

```diff
@@ -0,0 +1,192 @@
+
+Most of the simulator backends (native, RealFlight,etc) support changing the speed while
+running (NOT Realflight). Just set the ``SIM_SPEEDUP`` parameter as needed. A value of 1
+means normal wall-clock time. A value of 5 means 5x realtime. A value
+of 0.1 means 1/10th of real time.
+
+Running with high-levels of speedup can result in enough network traffic that MAVProxy can’t keep up. An error such as “Set RC override timeout” or the vehicle entering GCS failsafe are typical of these failures. Reducing the ``SIM_SPEEDUP`` factor is typically sufficient to avoid this problem. Also, using the ``--udp`` options will use UDP instead of the slower TCP communication protocol and can help avoid this issue at higher speedups.
+
+Loading a parameter set
+-----------------------
+
+When starting SITL the first time, the device will be configured with
+default parameters for the vehicle/frame type. Typically, you will want to replace these with
+values that simulate the desired vehicle and environment conditions.
+
+::
+
+    param load <<path to parameter file>
+
+the working directory is the directory in which MAVProxy starts, not the simulation
+
+You can re-load the parameters later if you choose, or revert to the
+default parameters by starting SITL (**sim_vehicle.py**) with the
+``-w`` flag.
+
+.. note:: You may have to load them again, after typing ``param fetch``, in order to load parameters that require enabling before presenting their parameter set.  And, in some cases, you may even need to restart SITL in order for some new parameters, such as output function changes, to take effect. This can be avoided if you add the parameter file during the start of SITL with the "--add-param-file=*pathtofile* "....ie: sim_vehicle.py -f flightaxis:192.168.x.x - -map - -console --add-param-file=*pathtofile* -w. This adds the param file as a default and then wipes any previous param changes that may exist in the simulation directory. 
+
+Parameters can also be saved. For example, to save the parameters into
+the present working directory you might do:
+
+::
+
+    param save ./myparams.parm
+
+.. note:: the "default" parameter set emulates the parameter default values contained in the firmware. These used at startup for parameters, UNLESS the user has previously changed them for a parameter(s), in which case those changed values are used. In SITL, these changes are stored in a file in the simulation startup directory named ``eeprom.bin``. The above commands loads user set values for the parameters, not the default values created at simulation start.
+
+Testing RC failsafe
+-------------------
+
+To test the behaviour of ArduPilot when you lose remote control (RC),
+set the parameter ``SIM_RC_FAIL=1``, as shown:
+
+::
+
+    param set SIM_RC_FAIL 1
+
+This simulates the complete loss of RC input. If you just want to
+simulate low throttle (below throttle failsafe level) then you can do
+that with the RC command:
+
+::
+
+    rc 3 900
```

Comment:
There's actually a `SIM_RC_FAIL` value of 2 which is used to simulate receiver-goes-to-bind-time-values.

Reply:
added....actually I misunderstood this when updating the RC_FAIL param....will fix that now